packages/@sanity/field/src/types/image/diff/ImagePreview.tsx

```diff
@@ -43,17 +103,20 @@ export function ImagePreview(props: ImagePreviewProps): React.ReactElement {
     .fit('max')
 
   const assetChanged = diff.fromValue?.asset?._ref !== diff.toValue?.asset?._ref
-  const imageWrapperClassName =
-    is === 'from' && assetChanged ? styles.imageWrapperChanged : styles.imageWrapper
+
   const metaAction = action === 'changed' ? undefined : action
 
   return (
```

Comment:
The image icons seems to be a bit cramped compared to what it used to be:

![image](https://user-images.githubusercontent.com/48200/124918357-cc00be80-dfa9-11eb-8f2a-1455a0415ac2.png)
vs
![image](https://user-images.githubusercontent.com/48200/124918391-d327cc80-dfa9-11eb-9c31-5f3bf02aa166.png)

The right hand side looks slightly _better_ in my opinion, but the left hand side are a bit too close to the icon?

Reply:
Looks like the container doesn't stretch properly, might be an issue fixed in the next PR, will look into it and include it here if that's the case

Reply:
was a padding issue combined with a hard px width for the arrow column. Fixing 😊